package.json

```diff
@@ -7,6 +7,7 @@
     "email": "nik@nikgraf.com",
     "url": "http://www.nikgraf.com"
   },
+  "draft-js-plugins": "virtrujakev/draft-js-plugins",
```

Comment:
I think this line shouldn't here?

Reply:
I was wondering the same thing myself, but I added it following the contributing guide: https://github.com/draft-js-plugins/draft-js-plugins/blob/a51d04b86205ce80c6886ffb3cf8aaffa05bf412/CONTRIBUTING.md?plain=1#L56 Please feel free to delete and commit the change or let me know if you'd like me to do so. Thanks!